prompts/opencode/ao-human/SKILL.md

```
@@ -0,0 +1,28 @@
+
+Find escalated stories/tasks by tag in Ralph PRD files.
+
+Default tag: `@human`
+
+**Steps**
+
+1) Parse args from user input:
+   - Optional `--tag <tag>` (default `@human`)
+
+2) Search in PRD/task artifacts (only include paths that exist):
+```bash
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)